packages/gestalt/src/icons/duplicate.svg

```diff
@@ -0,0 +1 @@
+<svg width="24px" height="24px" xmlns="http://www.w3.org/2000/svg"><path d="M20.1818182,9.54545455 C20.1818182,8.79272727 19.5709091,8.18181818 18.8181818,8.18181818 L15.8181818,8.18181818 L15.8181818,5.18181818 C15.8181818,4.42870261 15.207661,3.81818182 14.4545455,3.81818182 C13.7014299,3.81818182 13.0909091,4.42870261 13.0909091,5.18181818 L13.0909091,8.18181818 L10.0909091,8.18181818 C9.33779352,8.18181818 8.72727273,8.79233898 8.72727273,9.54545455 C8.72727273,10.2985701 9.33779352,10.9090909 10.0909091,10.9090909 L13.0909091,10.9090909 L13.0909091,13.9090909 C13.0909091,14.6622065 13.7014299,15.2727273 14.4545455,15.2727273 C15.207661,15.2727273 15.8181818,14.6622065 15.8181818,13.9090909 L15.8181818,10.9090909 L18.8181818,10.9090909 C19.5709091,10.9090909 20.1818182,10.2981818 20.1818182,9.54545455 M24,2.45454545 L24,16.6363636 C24,17.9919717 22.9010626,19.0909091 21.5454545,19.0909091 L7.36363636,19.0909091 C6.00802834,19.0909091 4.90909091,17.9919717 4.90909091,16.6363636 L4.90909091,2.45454545 C4.90909091,1.09893743 6.00802834,2.42230478e-16 7.36363636,0 L21.5454545,0 C22.9010626,0 24,1.09893743 24,2.45454545 M17.4545455,22.6363636 C17.4545455,23.3890909 16.8436364,24 16.0909091,24 L6.81818182,24 C3.05434854,23.9957916 0.00420842706,20.9456515 0,17.1818182 L0,7.90909091 C2.42230478e-16,7.15597534 0.610520796,6.54545455 1.36363636,6.54545455 C2.11675193,6.54545455 2.72727273,7.15597534 2.72727273,7.90909091 L2.72727273,17.1818182 C2.72967779,19.4401679 4.55983207,21.2703222 6.81818182,21.2727273 L16.0909091,21.2727273 C16.8436364,21.2727273 17.4545455,21.8836364 17.4545455,22.6363636" fill="#8E8E8E" fill-rule="nonzero"/></svg>
```

Comment:
@rootvar are these from production design team? They look like they were not optimized
![image](https://user-images.githubusercontent.com/7877010/56250537-97b92480-6064-11e9-91bd-466c32f06798.png)

Reply:
They are from our designer (exported from Sketch). Do we use svgo to optimize svgs?

Reply:
Yup, but you shouldn’t need to do anything. Can you or your designer please ping @jaymarsh or @scottandreae with the icon requests? They will create the optimized pds approved asset and those are the ones that go into gestalt (we don’t use ones from sketch). There’s a specific form for your designer to fill out under the PD project in Jira